src/Button/index.js

```diff
@@ -24,6 +24,10 @@ const Button = ({
   component,
   block,
   large,
+  // Types have been used for styling so we need another prop to specify submit buttons
```

Comment:
I've been meaning to introduce a breaking change here to use `variants` like other components. Thoughts?

Reply:
Sounds good. I'll switch to variants. I'll also add a forwardRef to Button since that will be a breaking change too.

Reply:
Just noticed that this is done is master. Will update the branch.